web/api/app/Config/bootstrap.php.in

```diff
@@ -101,15 +101,17 @@ CakeLog::config('debug', array(
 	'engine' => 'File',
 	'types' => array('notice', 'info', 'debug'),
 	'file' => 'cake_debug',
+    'path' => '@ZM_LOGDIR@/'
 ));
 CakeLog::config('error', array(
 	'engine' => 'File',
 	'types' => array('warning', 'error', 'critical', 'alert', 'emergency'),
 	'file' => 'cake_error',
+    'path' => '@ZM_LOGDIR@/'
 ));
 CakeLog::config('custom_path', array(
     'engine' => 'File',
-    'path' => '@ZM_LOGDIR@'
+    'path' => '@ZM_LOGDIR@/'
```

Comment:
Perhaps the only change required is the trailing slash here at this line. Did you check whether the other two path entries are really required? It doesn't seem like that should be necessary.